src/queue.ts

```diff
@@ -31,7 +32,7 @@ export class Queue<TItem> extends EventEmitter {
     }).promise()
   }
 
-  startProcessing (handler : (item : TItem) => any | PromiseLike<any>, options?: ProcessOptions): PromiseLike<void> {
+  startProcessing (handler : (item : TItem, message : any) => any | PromiseLike<any>, options?: ProcessOptions): PromiseLike<void> {
```

Comment:
@grvcoelho Here is the file we need. https://github.com/ingenieux/aws-sdk-typescript/blob/abd74761bdc12700f46dffa48a5811159d9bd096/output/typings/aws-sqs.d.ts

It's necessary to import the typings of the project, and change the type of the parameter.

Even though it's not this library's goal, it's provides the best practice for a TypeScript library.